example/app/app.component.scss

```diff
@@ -0,0 +1,14 @@
+.fa-check-circle-success {
+  color: green;
+}
+
+.fa-exclamation-triangle-warning {
```

Comment:
you can use `.fa-exclamation-triangle.warning` instead, then html class will be shorter,
`fa-exclamation-triangle fa-exclamation-triangle-warning` to `fa-exclamation-triangle warning`